main.go

```diff
@@ -55,15 +55,15 @@ func installDependency(packageManager jsdependency.Tool, name string, version st
 	}
 	for i, cmd := range cmdSlice {
 		fmt.Println()
-		log.Donef("$ %s", cmd.PrintableCommandArgs())
+		log.Donef("$ %s", cmd.Slice.PrintableCommandArgs())
```

Comment:
In [line#52](https://github.com/bitrise-steplib/steps-cordova-archive/pull/47/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R52) `jsdependency.InstallGlobalDependencyCommand` now returns a custom object, containing the model instance, stored in the `Slice` attribute.